credsweeper/credentials/line_data.py

```diff
@@ -139,7 +139,7 @@ def set_pattern_match_groups(self) -> None:
         if match_obj is None:
             return
 
-        def get_group_from_match_obj(match_obj: regex.Match, group: str) -> Optional[str]:
+        def get_group_from_match_obj(match_obj: regex.Match, group: str) -> Any:
```

Comment:
Why not `Optional[str]`?
https://stackoverflow.com/a/39429578

Reply:
credsweeper/credentials/line_data.py:144: error: Returning Any from function declared to return "Optional[str]"
it is not 'str' but AnyStr

Reply:
So why not to specify `AnyStr`? 
[AnyStr](https://docs.python.org/3/library/typing.html#typing.AnyStr)

`Any` is just too generic to provide any meaningful information